public/javascripts/detailedResults.js

```diff
@@ -0,0 +1,36 @@
+const playSessionData = JSON.parse(
+  document.getElementById("results-data").value
+);
+
+//image slides for each game (initiated and finished up with buttons)
+let slides = `<div class="button-div prev" onclick="plusSlides(-1)">&#10094;</div>`;
+//dots for another option to scroll through slides, only present if there are fewer than 20 slides
+let dots = "";
+//details for each game
+let gameInfo = "";
+
+//generating image slides and game info for each entry
+for (let i = 0; i < playSessionData.length; i++) {
```

Comment:
I would recommend to use array method `forEach`

Reply:
I still tend to default to a `for` loop, will change that